datasource/src/test/scala/astraea/spark/rasterframes/datasource/rastersource/RasterSourceDataSourceSpec.scala

```diff
@@ -108,6 +113,16 @@ class RasterSourceDataSourceSpec extends TestEnvironment with TestData {
       df.schema.size should be(2)
     }
 
+    it("should read a multiple files from a table") {
+      Seq(cogPath, l8samplePath, nonCogPath).toDF("path").createOrReplaceTempView("pathsTable")
+      val df = spark.read.rastersource
+        .fromTable("pathsTable", "path")
+        .withTileDimensions(128, 128)
+        .load()
+      df.select("path").distinct().count() should be(3)
+      df.schema.size should be(2)
```

Comment:
What would happen if the imagery is multi-band? Supported? Get multiple tile columns?

Reply:
By default it only reads the first band. If you want other bands you have to use the function `withBandIndexes(bandIndexes: Int*)`.